### ISSUE TITLE
[`rTexPacker`] Sprite Origin issue

### ISSUE THREAD

**Issue by mgobell** [raylibtech/rtools#44]
Hello!

I was playing around with your rTexPacker tool on itch.io, and noticed an issue with updating the sprite origin. After updating the sprite origin, whenever you try to select a different tile, the initial sprite origin is updated to the location you clicked on the new tile (which is under the sprite view window). This also happens when you click on the sprite origin button on the bottom of the sprite view window.

Basically, the origin is updated to locations you click on under the sprite origin window. So after you update a sprite origin, it's not possible to save that location because it updates to a different location after interacting with the UI.

**Comment by raysan5** [comment]
@mgobell Thanks for reporting. I'm planning a big update of rTexPacker in the following days and that issue should be addressed.

**Comment by raysan5** [comment]
Issue reviewed on latest rTexPacker 4.0 release: https://raylibtech.itch.io/rtexpacker
